spec/models/event_spec.rb

```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe Event, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  describe "validations" do
+    it "requires an unique meetup_url" do
+      e = Event.create(meetup_url:"www.meetup.com")
+      e2 = Event.new(meetup_url:"www.meetup.com")
+      expect(e2).to be_invalid
```

Comment:
Please improve this to look for the specific error instead of just being invalid